src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Class to represent the task in the task list.
+ */
+public class Task {
+
+    protected final String job;
+    protected String status;
+
+    /**
+     * Constructs a new task.
+     * @param job the task to be added to the task list.
+     */
+    public Task(String job) {
+        this.job = job;
+        this.status = "[✗]";
+    }
+
+    /**
+     * Changes the status of the task to done.
+     */
+    public void isDone() {
+        this.status = "[✓]";
+    }
+
+    /**
+     * Find the job of the task.
```

Comment:
For getters and setters, a Javadoc may not be required.
From the Google Java Style Guide (listed on CS2103T Java Coding Standard Website)"
![image](https://user-images.githubusercontent.com/39525563/64270640-78b52380-cf6e-11e9-85da-718b9f205f35.png)